src/main/java/hudson/tasks/Mailer.java

```diff
@@ -687,22 +702,16 @@ public FormValidation doSendTestMail(
                 @QueryParameter boolean useSsl, @QueryParameter boolean useTls, @QueryParameter String smtpPort, @QueryParameter String charset,
                 @QueryParameter String sendTestMailTo) throws IOException {
             try {
-                // TODO 1.590+ Jenkins.getActiveInstance
-                final Jenkins jenkins = Jenkins.getInstance();
-                if (jenkins == null) {
-                    throw new IOException("Jenkins instance is not ready");
-                }
+                Jenkins.get().checkPermission(DescriptorImpl.getJenkinsManageOrAdmin());
 
-                jenkins.checkPermission(Jenkins.ADMINISTER);
```

Comment:
I'm not sure we want to relax this.

Reply:
how so - you are allowing a user who has Manage to check what they have entered is correct.  Can you explain how they could use this check to escallate to `Overall.ADMINISTER`?

Reply:
No sorry I misunderstood the new permission.